include/odp/api/spec/packet.h

```diff
@@ -50,29 +50,22 @@ extern "C" {
  * Invalid packet segment
  */
 
- /**
-  * @typedef odp_packet_color_t
-  * Color of packet for shaper/drop processing
-  */
-
- /**
-  * @def ODP_PACKET_GREEN
-  * Packet is green
-  */
-
- /**
-  * @def ODP_PACKET_YELLOW
-  * Packet is yellow
-  */
-
- /**
-  * @def ODP_PACKET_RED
-  * Packet is red
-  */
-
 /**
- * Maximum number of packet colors which accommodates ODP_PACKET_GREEN, ODP_PACKET_YELLOW and
- * ODP_PACKET_RED.
+ * @enum odp_packet_color_t
+ * Color of packet for shaper/drop processing
+ *
+ * @var ODP_PACKET_GREEN
+ * Packet is green
+ *
+ * @var ODP_PACKET_YELLOW
+ * Packet is yellow
+ *
+ * @var ODP_PACKET_RED
+ * Packet is red
+ *
+ * @def ODP_NUM_PACKET_COLORS
```

Comment:
This ODP_NUM_PACKET_COLORS define is different from the packet_color_t enum. So, enum+vars should be one doxygen comment block and ODP_NUM_PACKET_COLORS another.

Reply:
Fixed.